src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColour(new ColorSupplier().getRandomColor().toString());
+        ball.setNumber(new Random().nextInt(100));

```

Comment:
Make ColorSupplier and Random variables a class like you did in the ColorSupplier class

Reply:
+